lib/validation.ts

```diff
@@ -339,10 +339,14 @@ export function validation(
     } else if (val.match(/^--no-.+/)) {
       // check if key/value doesn't exist
       val = val.match(/^--no-(.+)/)[1];
-      val = !Object.prototype.hasOwnProperty.call(argv, val);
+      val =
+        !Object.prototype.hasOwnProperty.call(argv, val) &&
+        argv[val] !== undefined;
```

Comment:
There is a negation on this one, so:
```suggestion
      val =
        !Object.prototype.hasOwnProperty.call(argv, val) ||
        argv[val] === undefined;
```

Reply:
Ah, good catch.